### PR TITLE
React form controls

### PR DESCRIPTION
**Pull request by Lee-SungWon**
Screen Shot 2020-09-18 at 9 07 08 AM

### CODE REVIEW

File: react-form-controls/src/index.jsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    event.preventDefault();
+  }
+
+  render() {
+    return (
+      <form onSubmit={this.handleSubmit}>
+        <label>
+          Email
+        </label>
+        <input type="email" onChange={this.handleChange}></input>
```
Comment (thebearingedge) [inline react-form-controls/src/index.jsx]: Be sure to also pass a `value` prop to your input so that it is fully "controlled".

**Review by thebearingedge — CHANGES_REQUESTED** [review]

File: react-form-controls/src/index.jsx (hunk 1 above)
Comment (Lee-SungWon) [reply react-form-controls/src/index.jsx]: revised!

Comment (Lee-SungWon) [reply react-form-controls/src/index.jsx]: I'm unclear as to exactly what value prop does though.  would you be able to clarify?

Comment (thebearingedge) [reply react-form-controls/src/index.jsx]: The idea is that React should know about and dictate the value in the input at any given time. This becomes more relevant in situations where you want to preload some initial form values, like if a user wants to edit something they had previously saved.

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**